src/components/Avatar/Avatar.tsx

```diff
@@ -89,24 +100,32 @@ class Avatar extends UIComponent<any, any> {
   }
 
   renderComponent({ ElementType, classes, rest }) {
-    const { src, alt, name, status } = this.props
+    const { src, alt, name, status, generateInitials } = this.props
     const { icon = '', color = '' } = Avatar.statusToIcon[status] || {}
+    const generateInitialsFunc = generateInitials || Avatar.generateInitials
```

Comment:
This is a good use case for `defaultProps`.  It will then also get typings and doc site generation.